pyproject.toml

```diff
@@ -28,6 +28,12 @@ per-file-ignores = {"tests/*" = ["D100","D101","D102","D103"]}
 convention = "google"
 
 # Static analysis tools configuration
+[tool.pyright]
+include = ["src"]
+extraPaths = ["lib"]
```

Comment:
I wonder how this interacts with `charm,lib` commands below:
```
    charm: pyright {[vars]src_path} {posargs}
    lib: pyright --pythonversion 3.5 {[vars]lib_path} {posargs}
```
- If you pass `{[vars]src_path}`, `{[vars]lib_path}`, do we still need `include` and `extraPaths`?
- What's the difference between `include = ["src"]` and `include = ["src", "lib"]`?
- For the `lib` env, should `include` be `lib/charms/alertmanager-k8s` and `extraPaths` blank?
- Could/should these two be provided as args instead of config options?